### PR TITLE
Use temporary folders and improve displayed paths

### PR DESCRIPTION
**Pull request by ybart**
Rework to use a temporary folder instead of creating files in parallel.
Add command line option to define a temporary folder for generated files
Passthrough some Rubocop switches (--auto-get-config, --config, ...)

〜 @derjanb

Update analyzer to fix paths for easier access

〜 @ybart

Exit rake on failure

〜 @ybart

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: ruby
 rvm:
-  - 2.2.2
+  - 2.4.1
```
Comment (ericqweinstein) [inline .travis.yml]: 💯

File: Rakefile [hunk 2]
```diff
@@ -30,8 +30,8 @@ namespace :test do
   task all: %i(spec rubocop)
 end
 
-desc 'Generate Ruby documentation'
-YARD::Rake::YardocTask.new do |t|
-  t.files   = %w(lib/**/*.rb)
-  t.options = %w(--protected --private)
-end
+# desc 'Generate Ruby documentation'
```
Comment (ericqweinstein) [inline Rakefile]: @ybart Feel free to remove

File: VERSION [hunk 3]
```diff
@@ -1 +1 @@
-0.0.1
+0.1.0
```
Comment (ericqweinstein) [inline VERSION]: 💯

File: lib/ruumba/rake_task.rb [hunk 4]
```diff
@@ -15,8 +15,9 @@ class RakeTask < Rake::TaskLib
     def initialize(*args, &block)
       @name = args.shift || :ruumba
       @dir  = []
+      @options = nil
 
-      desc 'Run RuboCop on ERB files' unless ::Rake.application.last_comment
+      desc 'Run RuboCop on ERB files' # unless ::Rake.application.last_comment
```
Comment (ericqweinstein) [inline lib/ruumba/rake_task.rb]: @ybart Feel free to remove the commented-out `unless ::Rake.application.last_comment`

**Comment by ybart** [comment]
Thanks for review, I fixed and added one more feature to one able to exit rake when ruumba fails. I needed  this feature for my CI.

**Comment by ericqweinstein** [comment]
@ybart Thanks! I'll deploy the new version of the gem later today.

**Comment by ybart** [comment]
Thanks for the merge and for the upcoming gem release !

**Comment by ericqweinstein** [comment]
@ybart Sure thing! Apologies for the delay, should be out tomorrow 👍

**Comment by ybart** [comment]
No problem ! Thanks !

**Pull request merged**